vnext/ReactWindowsCore/TurboModuleManager.cpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include "TurboModuleManager.h"
+#include "turbomodule/samples/SampleTurboCxxModule.h"
+
+namespace facebook {
+namespace react {
+
+TurboModuleManager::TurboModuleManager(
+    std::shared_ptr<TurboModuleRegistry> turboModuleRegistry,
+    std::shared_ptr<JSCallInvoker> jsInvoker)
+    : m_turboModuleRegistry(turboModuleRegistry), m_jsInvoker(jsInvoker) {}
+
+/**
+ * Return the TurboModule instance that has that name `moduleName`. If the `moduleName`
+ * TurboModule hasn't been instantiated, instantiate it. If no TurboModule is registered under
+ * `moduleName`, return null.
+ */
+std::shared_ptr<TurboModule> TurboModuleManager::getModule(const std::string &moduleName) {
+  if (!hasModule(moduleName)) {
```

Comment:
>hasModule [](start = 7, length = 9)

Use of this API causes making search twice.
It would be better to examine iterator here.